news/PR-0556.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:**
+
+* Use bind(c) in fmesh_mod.F90 to avoid the need for name mangling on the C++ side
```

Comment:
Can you add renaming of the patch files here? This will be important for external users.

Reply:
Good call. Done.